partner_communication_switzerland/models/contracts.py

```diff
@@ -88,13 +88,6 @@ def _do_not_send_letter_to_transfer(self):
         else:
             self.send_introduction_letter = True
 
-    @api.multi
-    def _on_change_correspondant(self, correspondent_id):
-        # Don't send introduction letter when correspondent is changed
-        cancelled_sponsorships = super()._on_change_correspondant(correspondent_id)
-        cancelled_sponsorships.write({"send_introduction_letter": False})
-        return cancelled_sponsorships
-
```

Comment:
I would still keep the onchange method so that by default it disables the setting, while the user still has the final control on the value he wants to put and can still re-activate it if needed.